gems/pending/util/miq_ree_gc.rb

```diff
@@ -1,33 +0,0 @@
-module MiqReeGc
```

Comment:
Wow, good find @matthewd, I forgot about that... you unearthed and deleted a ruby 1.8.7 relic.  :100: